clang/test/CXX/drs/dr0xx.cpp

```diff
@@ -80,14 +86,21 @@ namespace dr5 { // dr5: 3.1
 
 namespace dr7 { // dr7: 3.4
   class A { public: ~A(); };
-  class B : virtual private A {}; // expected-note 2 {{declared private here}}
-  class C : public B {} c; // expected-error 2 {{inherited virtual base class 'A' has private destructor}} \
-                           // expected-note {{implicit default constructor for 'dr7::C' first required here}} \
-                           // expected-note {{implicit destructor for 'dr7::C' first required here}}
+  class B : virtual private A {}; // #dr7-B
+  class C : public B {} c; // #dr7-C
+  // expected-error@#dr7-C {{inherited virtual base class 'A' has private destructor}}
+  // expected-note@#dr7-C {{in implicit default constructor for 'dr7::C' first required here}}
+  // expected-note@#dr7-B {{declared private here}}
```

Comment:
I have to say, it is kind of messing with me that we have `-C` before `-B` but I then realized that `B` and `C` are the class names here and don't imply some sort of ordering.

I wanted to point that out in case others also felt that way.

Reply:
I had a thought of naming those markers `dr7-struct-C`, but decided to prioritize shorter markers. If you find this more accessible, we can do it this way.

Reply:
I thought that lower case vs upper case makes enough distinction, but I don't insist.

Reply:
I think the shorter markers are OK.  This is already vastly more readable, and we can blame CWG for making the tokens confusing :)

Reply:
This is not a sticking point for me, just an observation and to see how others felt.